src/main/resources/templates/index.html

```diff
@@ -14,7 +14,7 @@
 </style>
 <body>
 <h1>WELCOME TO SURVEY MONKEY </h1>
-
+<!Setting up buttons on home page and an image>
```

Comment:
This is invalid html comment. HTML comments are done like so https://www.w3schools.com/html/html_comments.asp

Reply:
copy, will change that now. Thank you for pointing that out